utils/vscode/LICENSE

```diff
@@ -0,0 +1,234 @@
+==============================================================================
```

Comment:
This is not necessary. Clone LICENSE file in the extension repository (circt/vscode-circt).